scss/bootstrap/overrides/partials/_modals-xs-screen.scss

```diff
@@ -0,0 +1,36 @@
+// Modal for XS screens such as smartphones
```

Comment:
I think this should go in the variants/ folder

Reply:
I actually had put it in variants first, then thought it is not really a variant... But OK.